main.tf

```diff
@@ -91,163 +98,107 @@ resource "helm_release" "gitlab-runner-linux" {
 }
 
 
-resource "helm_release" "gitlab-runner-win" {
+resource "helm_release" "gitlab-runner-kaniko" {
   depends_on = [
                 module.gke,
-                module.gke_auth
+                module.gke_auth,
+                kubernetes_namespace.sensitive-namespace
                 ]
-  name       = "windows"
-  # repository = "https://charts.gitlab.io"
-  chart      = "https://gitlab.com/gitlab-org/charts/gitlab-runner/-/archive/v0.32.0/gitlab-runner-v0.32.0.tar.gz"
+  name       = "kaniko"
+  wait       = false
+  namespace  = "sensitive"
+  # repository = "https://charts.gitlab.io/gitlab"
+  chart      = "https://gitlab-charts.s3.amazonaws.com/gitlab-runner-0.33.1.tgz"
   
   values     = [
-    file("${path.module}/gitlab-runner/win-values.yaml")
+    file("${path.module}/gitlab-runner/kaniko-values.yaml")
     ]
 
   set {
     name  = "gitlabUrl"
-    value = "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
   set {
-    name  = "cloneUrl"
-    value = "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
+    name  = "runners.cloneUrl"
+    value =  "${var.gitlab_instance_protocol}://${local.instance_internal_domain}"
   }
   set {
     name  = "certsSecretName"
     value = "${local.instance_internal_domain}-cert"
   }
   set_sensitive {
     name  = "runnerRegistrationToken"
-    value = random_password.gitlab_runner_registration_token.result
+    value = "${random_password.gitlab_runner_registration_token.result}-scallops-recipes"
   }
 }
 
-########################### GKE Cluster #################################################
-
+resource "helm_release" "gitlab-runner-dockerhub" {
```

Comment:
Why didn't you move this to gke file?

Reply:
Although it is deployed to the GKE I still consider it as a different component that connects the gitlab to GKE.
I thought of putting it there but it will make the gke.tf heavier and leave our main.tf only with gitlab instance

Reply:
I think we need to move, otherwise it might confuse.
Don't worry if the main will be lean.

Reply:
What about adding 'helm.tf' and rename 'main.tf' to 'gitlab.tf'?

Reply:
GKE already stuffed with many things